test/threads.jl

```diff
@@ -518,3 +532,9 @@ let e = Event(), started = Event()
     wait(@async (wait(e); blocked = false))
     @test !blocked
 end
+
+@testset "bad arguments to @threads" begin
+    @test_me ArgumentError @macroexpand(@threads 1 2) # wrong number of args
+    @test_me ArgumentError @macroexpand(@threads 1) # arg isn't an Expr
+    @test_me ArgumentError @macroexpand(@threads if true 1 end) # arg doesn't start with for
```

Comment:
Care to remake this? Looks like this can now be just:
```suggestion
    @test_throws ArgumentError @macroexpand(@threads 1 2) # wrong number of args
    @test_throws ArgumentError @macroexpand(@threads 1) # arg isn't an Expr
    @test_throws ArgumentError @macroexpand(@threads if true 1 end) # arg doesn't start with for
```